plugins/jspsych-spr-moving-window.js

```
@@ -581,3 +556,44 @@ var sprMovingWindow = (function(jspsych) {
     return SprMovingWindowPlugin;
 
 })(jsPsychModule);
+
+/**
+ * Parses a stimulus and returns a GroupList that contains the
+ * groups of words that should be presented simultaneously in the spr.
+ *
+ */
+function parseSpr(stimulus) {
+    const parser = new nearley.Parser(nearley.Grammar.fromCompiled(spr_grammar))
+    parser.feed(stimulus)
+    let tree = parser.results
+    if (tree.length > 1) {
+        console.error("The grammar is ambigious for this stimulus");
```

maybe this should throw an error instead?